tests/jenkins/nrnivmodl.sh

```diff
@@ -6,17 +6,10 @@ set +x
 TEST_DIR="$1"
 
 . /gpfs/bbp.cscs.ch/apps/hpc/jenkins/config/modules.sh
-module load intel
+export SPACK_INSTALL_PREFIX="${SPACK_INSTALL_PREFIX:-${WORKSPACE}/INSTALL_HOME}"
+export MODULEPATH=$SPACK_INSTALL_PREFIX/modules/tcl/linux-rhel7-x86_64:$MODULEPATH
```

Comment:
`linux-rhel7-x86_64` this will change with OS change. I don't remember command to get modules directory but could you check in spack issues?

Reply:
I tried looking for this but I didn't find anything. Another way I found to solve this is to use `$(spack arch)` instead of `linux-rhel7-x86_64`, similarly to what is done in other jenkins plans